tests/integration/sync/it_sync_location_test.py

```diff
@@ -0,0 +1,88 @@
+import unittest
+
+import pytest
+from six import unichr
+
+from tests.poster_mock import free_port, start_server
+from timeless.companies.models import Company
+from timeless.restaurants.models import Location
+from timeless.sync.synced_location import SyncedLocation
+from timeless.poster.api import Poster
+
+"""Integration tests for Location Sync with database"""
+
+
+"""
+    Tests for poster sync with database.
+
+"""
```

Comment:
@paulodamaso I think only the first docstring is enough, this one can be removed